.zuul.yaml

```diff
@@ -28,7 +28,12 @@
                   service_available.swift false
                   service_available.cinder false
               cifmw_run_tobiko: true
-              cifmw_test_operator_tobiko_testenv: 'functional -- tobiko/tests/functional/podified/test_topology.py'
+              cifmw_test_operator_tobiko_workflow:
+                - stepName: 'podified-functional'
+                  testenv: 'functional -- tobiko/tests/functional/podified/test_topology.py'
```

Comment:
```
cp: target '/var/lib/tobiko/external_files/tobiko-tobiko-tests-workflow-step-0/' is not a directory
```
The issue is probably the `--` in the name of the testenv [1].  

[1] https://logserver.rdoproject.org/61/61/f6d4ad5e7af4032384e64f28b33a22a0fae53b02/github-check/podified-multinode-edpm-deployment-crc/a78b181/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/tobiko-tests-podified-functional-workflow-step-0-sn6v6/logs/tobiko-tests.log

Reply:
Posible fix? -> https://github.com/openstack-k8s-operators/tcib/pull/153